grammar.js

```diff
@@ -75,6 +75,7 @@ module.exports = grammar({
     keyword_cross: _ => make_keyword("cross"),
     keyword_join: _ => make_keyword("join"),
     keyword_lateral: _ => make_keyword("lateral"),
+    keyword_natural: _ => make_keyword("natural"),
```

Comment:
please add this keyword also to the hightlights.scm

Reply:
under `@keyword` would be a good place (next to `keyword_lateral` for example)

Reply:
Done.